frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/os-flavor.tsx

```diff
@@ -91,28 +105,90 @@ export const OSFlavor: React.FC<OSFlavorProps> = React.memo(
       }
     }
 
+    const loadedBaseImages = iGetLoadedData(openshiftCNVBaseImages);
+    const operatingSystemBaseImages = operatingSystems.map(
+      (operatingSystem: OperatingSystemRecord) => {
+        const pvcName = operatingSystem?.dataVolumeName;
+        const imageFoundInCluster = loadedBaseImages?.find(
+          (dataVolume) => iGetName(dataVolume) === pvcName,
+        );
+
+        return {
+          id: operatingSystem.id,
+          name: operatingSystem.name,
+          pvcName,
+          imageFoundInCluster,
+          message:
+            (!pvcName && '(Missing Disk Image)') ||
+            (pvcName && !imageFoundInCluster && '(Missing Disk Image)') ||
+            '',
+          longMessage:
+            (!pvcName &&
+              'The Operating System Template is missing a disk image definitions, a custom boot source must be definfed manually') ||
```

Comment:
grammar

The Operating System Template is missing disk image definitions, a custom boot source must be defined manually

Reply:
wow Thanks :exclamation: